build.gradle

```diff
@@ -662,9 +671,9 @@ task runJava(type: JavaExec) {
 
     standardOutput = System.out
     errorOutput = System.err
-
+    executable = '/Applications/MagicDraw/cae-cameo-systems-modeler-5.0.0-SNAPSHOT-macos/jre/Contents/Home/bin/java'
```

Comment:
Again, not available in public bundle so should stick to using the `java` executable in system path.